stl/inc/coroutine

```diff
@@ -10,12 +10,14 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #ifdef _RESUMABLE_FUNCTIONS_SUPPORTED
-#pragma message("The contents of <coroutine> are not available with /await.")
-#pragma message("Remove /await or use /await:strict for standard coroutines.")
-#pragma message("Use <experimental/coroutine> for legacy /await support.")
+#pragma message(_WARNING_MESSAGE("STL4038", "The contents of <coroutine> are not available with /await."))
+#pragma message(_WARNING_MESSAGE("STL4038", "Remove /await or use /await:strict for standard coroutines."))
+#pragma message(_WARNING_MESSAGE("STL4038", "Use <experimental/coroutine> for legacy /await support."))

```

Comment:
All of the other warnings are about needing a newer Standard mode, but this one is different - it's complaining that the Standard header isn't compatible with the legacy switch. We should use a different warning number for this.

Additionally, if we have to touch this code, it would be better to emit a single message; currently this emits

```
D:\GitHub\STL\out\build\x64\out\inc\coroutine(13): warning STL4038: The contents of <coroutine> are not available with /await.
D:\GitHub\STL\out\build\x64\out\inc\coroutine(14): warning STL4038: Remove /await or use /await:strict for standard coroutines.
D:\GitHub\STL\out\build\x64\out\inc\coroutine(15): warning STL4038: Use <experimental/coroutine> for legacy /await support.
```

but this is different than how compiler warnings behave (the compiler will print additional "note" lines, but not repeat the warning number; I don't think we need to imitate the "note" part though).